src/components/gig/signup-box.tsx

```diff
@@ -30,6 +30,7 @@ interface GigSignupBoxProps {
   checkbox1: string;
   checkbox2: string;
   signup?: Signup;
+  signupEndExsists?: boolean;
```

Comment:
I think we should send the options as a prop rather than sending `signupEndExists`.